README.md

```diff
@@ -1,3 +1,45 @@
 # Bootcamp
-hii 
-hello world
\ No newline at end of file
+SQL Server Query Script
+
+This Python script connects to a SQL Server database (AdventureWorksDW2022) using pyodbc and performs some basic SQL queries on the DimEmployee table.
+
+Requirements
+Python 3.6+: Install Python from python.org.
+
+pyodbc: Python library for SQL Server connection.
+
+SQL Server ODBC Driver: Required to connect Python to SQL Server.
+
+Install Dependencies
+To install the required Python package (pyodbc), run the following command:
+
+pip install pyodbc
```

Comment:
I would like to see requirements.txt file for all the pip install that I need. Also please use 3.11+ and also start using virtu
al environment